kedro/framework/project/__init__.py

```diff
@@ -195,6 +198,13 @@ def configure_project(package_name: str):
     PACKAGE_NAME = package_name
 
 
+def configure_logging(conf_logging):
```

Comment:
Can we add some typehints here so we know what to expect?
```suggestion
def configure_logging(conf_logging: Dict[str, Any]) -> None:
```

not sure if the argument is Dict[str, Any] or Dict[str, str]

Reply:
Also, this is a minor thing, but can we think of a better name than `conf_logging`? 😅 config, configuration, logging_config etc. sound better too me but it completely optional.